go.mod

```diff
@@ -3,9 +3,17 @@ module github.com/openstack-k8s-operators/osp-director-operator
 go 1.13
 
 require (
+	github.com/Masterminds/sprig v2.22.0+incompatible
 	github.com/go-logr/logr v0.1.0
 	github.com/onsi/ginkgo v1.12.1
 	github.com/onsi/gomega v1.10.1
+	github.com/openshift/api v3.9.0+incompatible
+	github.com/openstack-k8s-operators/lib-common v0.0.0-20201012132655-247b83b2fafa
+	github.com/openstack-k8s-operators/nova-operator v0.0.0-20201020141434-6a40ed962c36
```

Comment:
do we require lib-common and nova-operator?

Reply:
no we don't, thats got added in an initial version and missed to remove them